controllers/utils/volumegroupcontent.go

```diff
@@ -221,6 +228,19 @@ func updateVGCStatusPVList(client client.Client, vgc *volumegroupv1.VolumeGroupC
 	return nil
 }
 
+func UpdateVGCStatusError(client client.Client, vgc *volumegroupv1.VolumeGroupContent, logger logr.Logger, message string) error {
+	err := retry.RetryOnConflict(retry.DefaultRetry, func() error {
+		vgc.Status.Error = &volumegroupv1.VolumeGroupError{Message: &message}
+		err := vgcRetryOnConflictFunc(client, vgc, logger)
+		return err
+	})
+	if err != nil {
```

Comment:
You can just return err, no need of the if

Reply:
done